latex2lambda.py

```diff
@@ -0,0 +1,41 @@
+from latex2sympy2 import latex2sympy
+from sympy import lambdify
+
+
+def latex2lambda(
+    tex: str,
+    variable_values={},
+    args=None,
+    modules=None,
+    printer=None,
+    use_imps=True,
+    dummify=False,
+):
+    if not args:
+        try:
+            """
+            Trying to parse a function of type "f(...) = <...>" with the function itself on <...>
+            """
+            var_tex, tex = tex.replace(" ", "").split("=")
+            args = var_tex.split("(")[1].replace(")", "").split(",")
+        except Exception:
+            raise Exception("No valid arguments provided.")
```

Comment:
Could be a named exception so that the library user can detect exceptions by type.
However, do we really want to do a `try ... catch` inside this part of the library? I'd be OK with throwing whatever comes, and it's up to the user of the function to realize if it gets some exception declared in the `tex` package, it's up to them to catch them and handle them.

Reply:
Well in this particular case, I think it is appropriate, let me explain why. 
The latex2sympy method expects the tex to be just a function (without the need to recognize the variables). However, when making a lambda function we must provide the variables on which the function depends (that is, `f(x,y,z,...)=`, so it would give a `lambda x,y,z,...: <...>`). In this sense, either we pass the tex as `f(x,y,z,...) = <...>`, or we pass the function `<...>` and a list of variables on the argument `args`. In the first case, the `lambdify` function can only receive the operand (`<...>`), and so it is required an initial parsing.

In this sense, we could parse this inputs directly on the function (as I did there), or we can just assume the user knows how it works, but some errors would be ambiguous.

Reply:
I'm not saying you should not parse it, what I'm saying is what do you do when you fail it.
So, since you wrote a `try .. catch` block, I'm assuming line 19 and 20 can throw exceptions. If those exceptions themselves already have a meaning, you're basically just obscuring it with your new raise of the unnamed exception. If those exceptions to not have a meaning, you're also not adding much meaning to them with something that cannot be "caught" in the callers code. Take this example.

Me, as a developer of the back-end for a web app, would like to have an exception that identifies that the user failed to give a meaningful `tex`/`var_tex`, and I could catch it to then produce a response to the front-end, like 400, but with a content like: "your function is not valid latex, why don you try doing something alike (bla bla bla)"

Reply:
Oh I'm sorry, i get it. For now the exception is very broad, but I will add a customized exception to specify what error really occurs. For now, as you said, it s just an unnamed exception without meaning, but it will have meaning (still WIP on finding the best way to define the custom exception).

Reply:
Yea, f7471cc53f713ad217c7ff6c9d948f6bf5d57ff4 seems to solve my main concern, now at least we know better than to say "woops".